lib/netconf.js

```diff
@@ -185,26 +185,23 @@ class Client extends EventEmitter {
     }
 
     /**
-     * Closes the SSH connection to netconf, emits close event
+     * Closes the SSH connection to netconf
      * @function close
      * @param  {Function} callback Callback function fired after netconf session ends
      */
-    close(callback){
+    close(callback) {
         const self = this;
 
         // send close-session request to netconf
         self.rpc('close-session', (err, reply) => {
-            // close out SSH tunnel
-            self.sshConn.end();
-            self.connected = false;
-            self.netconf = void 0;
```

Comment:
We still need to keep these lines in, this closes the SSH tunnel, and sets the class state to disconnected

Reply:
Retained the two lines. Unfortunately I could not retain the self.netconf = void 0; since the code will throw an error at runtime because it needs netconf variable.